.storybook/webpack.config.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+
+module.exports = (baseConfig, env, defaultConfig) => {
+  defaultConfig.resolve.extensions.push('.jsx');
```

Comment:
We should be immutable rather than mutating `defaultConfig`.

Reply:
I was just going with the recommended way of extending the WebPack config in the Storybook docs https://storybook.js.org/configurations/custom-webpack-config/#full-control-mode--default I can change it to be immutable, though.